addons/binding/org.openhab.binding.co2signal/src/main/java/org/openhab/binding/co2signal/CO2SignalBindingConstants.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.co2signal;
+
+import java.util.Set;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+import com.google.common.collect.ImmutableSet;
```

Comment:
Can you use the following solution its prefered over the Guava dependency:

https://github.com/openhab/openhab2-addons/blob/9b0904c81e5c76f43f04d7d31ecb88c095cdb7bf/addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/BindingConstants.java#L101

Reply:
Fixed